website/projects/spoke/src/app/core/state/page.state.ts

```diff
@@ -0,0 +1,33 @@
+import { Injectable } from '@angular/core';
+import { DataAction, StateRepository } from '@ngxs-labs/data/decorators';
+import { NgxsImmutableDataRepository } from '@ngxs-labs/data/repositories';
+import { State } from '@ngxs/store';
+
+interface PageStateModel {
+  allowTelemetry: boolean;
```

Comment:
Change allowTelemetry to a tri-state boolean: ie `allowTelemetry: boolean | undefined` when `undefined` the user hasn't chosen whether to allow telemetry or not and the snackbar should stay up until they press one or the other. When telemetry is chosen, always show the snackbar for 3 seconds and then hide (currently its 10s I believe).